.travis.yml

```diff
@@ -1,25 +1,31 @@
-sudo: required
 language: go
 
+matrix:
+  include:
+    - os: linux
+      env: TEST_SUITE=test-unit
+    - os: linux
+      env: TEST_SUITE=test-bogo
+    - os: linux
+      env: TEST_SUITE=test-interop
+    # Travis does not support Docker on macOS.
+    - os: osx
+      env: TEST_SUITE=test-unit
+  fast_finish: true
+
 services:
   - docker
 
 go:
   - 1.11.x
 
-env:
-  - TEST_SUITE=test-unit
-  - TEST_SUITE=test-bogo
-  - TEST_SUITE=test-interop
-
-matrix:
-  fast_finish: true
-
 before_install:
   - make -f _dev/Makefile fmtcheck
 
 install:
-  - sudo pip install docker
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo pip install docker; fi
 
 script:
+  # Travis does not support Docker on macOS, disable building these.
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export DOCKER='echo Unsupported - docker'; fi
```

Comment:
I don't think  this is needed as you are running "test-unit" target only

Reply:
Unfortunately it is needed due to build-all -> build-test-tris-client -> `$(DOCKER) build ...`:

https://github.com/cloudflare/tls-tris/blob/master/_dev/Makefile#L91-L93

Reply:
I've added a comment to clarify this.